src/main/java/seedu/duke/commands/ShowRateCommand.java

```diff
@@ -59,18 +59,19 @@ public void execute(Ui ui, AccountList accounts) {
     }
 
     /**
-    * Prints the exchange rate between two currencies with a specified amount
-    * @param instance a Forex object containing the exchange rate
-    * @param amt a float of the amount to be converted on the exchange rate
-    * @return a string containing the exchange rates to be printed
-    * @throws InvalidNumberException if the amount is negative
-    */
-    private String getRateString(Forex instance, float amt) throws InvalidNumberException {
+     * Prints the exchange rate between two currencies with a specified amount
+     *
+     * @param instance a Forex object containing the exchange rate
+     * @param amt      a float of the amount to be converted on the exchange rate
```

Comment:
Remove unnecessary spaces
```suggestion
     * @param amt a float of the amount to be converted on the exchange rate
```

Reply:
This is a symptom of the formatter